content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -27,9 +27,35 @@ repo here](https://github.com/kubernetes/website/tree/{{< param "docsbranch" >}}
 
 {{% capture body %}}
 
+## nodeName
+
+`nodeName` is the simplest node selection constraint.
```

Comment:
I would point out that this is not a recommended way and move it after recommended ways (not before them).

Reply:
Done